documentation/features.asciidoc

```diff
@@ -12,11 +12,11 @@ Set up your IDE within minutes tailored for the requirements of your project.
 * *Automated* +
 Automate the setup and update, avoid manual steps and mistakes.
 * *Simple* +
-KISS (Keep It Small and Simple), no native installers that globally mess your OS or tool-integrations that break with every release. Instead, use templates and simple shell scripts.
+KISS (Keep It Small and Simple), no native installers that globally mess your OS or tool-integration that break with every release. Instead, use templates and simple shell scripts.
 * *Configurable* +
 You can change the link:configuration.asciidoc[configuration] depending on your needs. Furthermore, the link:settings.asciidoc[settings] contain configuration templates for the different tools (see link:configurator.asciidoc[configurator]).
 * *Maintainable* +
-For your project you should copy these link:settings.asciidoc[settings] to an own `git` repository that can be maintained and updated to manage the tool configurations during the project lifecycle. If you use github or gitlab every developer can easily suggest changes and improvements to these link:settings.asciidoc[settings] via pull/merge requests, which is easier to manage with big teams.
+For your project you should copy these link:settings.asciidoc[settings] to an own `git` repository that can be maintained and updated to manage the tool configurations during the project lifecycle. If you use `github` or `gitlab` every developer can easily suggest changes and improvements to these link:settings.asciidoc[settings] via pull/merge requests, which is easier to manage with big teams.
```

Comment:
git ,github,gitlab ->dict

Reply:
Please check what is the correct spelling for all three e.g. is it github or GitHub or Github,...?